### DISCUSSION TITLE
Cake Day and V1.6.0

### DISCUSSION THREAD

**Discussion by riffnshred** [riffnshred/nhl-led-scoreboard#222] [Announcements]
**FIRST**,  today one year ago, V1.0.0 was released. :beer: :confetti_ball: :partying_face: :tada: 
I want to thank everyone for being part of this project which made 2020 a bit more easy to go through. I hope you all are safe and sound that even tho your team suck, you are still happy for hockey to be going. I know I'm !!!

Second. There will me some changes coming for how the community will work. Github came up with a new [Discussions](https://github.com/riffnshred/nhl-led-scoreboard/discussions) section where communities can interact in a Forum kind of way. There is some nice features in there and  this is where I will post about the project from now on. This is because a lot of information get lost in the chat and especially when it comes to troubleshooting, we repeat some things too often.  DON'T WORRY, I'm keeping the Discord channel for chat and stuff.

**IMPORTANT** From now on, any help and troubleshooting will have to be done using the Issues section on github. https://github.com/riffnshred/nhl-led-scoreboard/issues. With the huge amount of people coming for help, especially in the beginning of this season, we (the moderators and my self and some core users) felt like we were running a tech support center. 

This part is more specifically for those that have less skills using terminals and how to run and install this project. I added some skills requirements for this project and they can be found on the documentation at the top.  Keep in mind that this is a **DIY project**, **not a product you bought at Bestbuy**. You have to be ready to do your own research and fail and learn some things. We wont take you by the hand to do each command to get to where you need to be. We document everything as much as possible specifically for the user to be able to run it. I'm open to modify the documentation if need be and if the request make sense.

**Third**. There is a new release coming out, NOW.  (It's a big one)
https://github.com/riffnshred/nhl-led-scoreboard/releases/tag/v1.6.0

About that new release: READ THE NEW DOCUMENTATION. A lot is changing, especially with the new image @falkyre#3190  has made. https://github.com/riffnshred/nhl-led-scoreboard

Every time I will make a new release, @falkyre#3190 will generate a new image and there will be a second announcement done automaticly when it's up on his repository here https://github.com/falkyre/nhl-led-scoreboard-img

**Comment by sflems** [comment]
Happy 🍰  day! 🍻

**Comment by riffnshred** [comment]
![birthdaygirl](https://media2.giphy.com/media/yoJC2GnSClbPOkV0eA/giphy.gif?cid=ecf05e475njst0g88f377hi8a0fe2esov4jtjgaqanv1obl1&rid=giphy.gif)

**Comment by jeepdrvr71** [comment]
Does the update procedure listed on here get you to 1.6 or do you need to install the image

**Reply by falkyre** [reply]
You can update with usual procedure as listed. If you want to start from scratch, the image has everything setup already and will get you the v1.6.0 code.
